lib/galaxy/datatypes/converters/CONVERTER_uncompressed_to_gz.xml

```diff
@@ -0,0 +1,24 @@
+<tool id="CONVERTER_uncompressed_to_gz" name="Compress file(s) using gzip" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.11">gzip</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        gzip -c ${input} > output_file.gz
+    ]]></command>
+    <inputs>
+        <param name="input" type="data" format="data" label="Select data to compress"/>
+    </inputs>
+    <outputs>
+        <data name="output_file" format="gz" from_work_dir="output_file.gz" label="GZIP compression on ${on_string}"/>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="1">
+            <param name="input" value="1.csv"/>
+            <output name="output_file" file="1.csv.gz" compare="sim_size"/>
```

Comment:
Please use an assertion instead of sim_size.

Reply:
Added a second test instead which decompresses and compares back to original

Reply:
We could avoid storing the test data file `1.csv.gz` by using:

```xml
<assert_contents>
    <has_size value="XXX" delta="YYY"/>
</assert_contents>
```